packages/amplify-category-api/resources/awscloudformation/cloudformation-templates/containers-template.json

```diff
@@ -0,0 +1,916 @@
+{
```

Comment:
This cloudformation template is going to have removed the VPC related resources.